config/environment.js

```diff
@@ -16,6 +16,7 @@ if (environment === 'development') {
   require('dotenv').config()
 }
 
+const vaultEndpoint = process.env.VAULT_ADDR || 'http://127.0.0.1:8200'
```

Comment:
add the `VAULT_ADDR` env variable to the chart values file https://github.com/godaddy/kubernetes-external-secrets/blob/master/charts/kubernetes-external-secrets/values.yaml#L6 and update README?

Reply:
Good point, adding this!